docs/source/quickstart.rst

```diff
@@ -3,6 +3,219 @@
+
+.. code-section::
+
+    .. code-block:: python
+
+        import mlflow
+
+        from sklearn.model_selection import train_test_split
+        from sklearn.datasets import load_diabetes
+        from sklearn.ensemble import RandomForestRegressor
+
+        mlflow.sklearn.autolog()
+
+        db = load_diabetes()
+        X_train, X_test, y_train, y_test = train_test_split(db.data, db.target)
+
+        # Create and train models.
+        rf = RandomForestRegressor(n_estimators=100, max_depth=6, max_features=3)
+        rf.fit(X_train, y_train)
+
+        # Use the model to make predictions on the test dataset.
+        predictions = rf.predict(X_test)
+
+In addition, or if you are using a library for which ``autolog`` is not yet supported, you may use key-value pairs to track:
+
+.. list-table::
+   :widths: 10 10 80 
+   :header-rows: 1
+
+   * - Name
+     - Used for
+     - Function call
+   * - Parameters
+     - Constant values (for instance, configuration parameters)
+     - ``mlflow.log_param``
+   * - Metrics
+     - Values updated during the run (for instance, accuracy)
+     - ``mlflow.log_metric``
+   * - Artifacts
+     - Files produced by the run (for instance, model weights)
+     - ``mlflow.log_artifacts``
+
+This example demonstrates the use of these functions:
+
+.. code-section::
+    .. code-block:: python
+
+        import os
+        from random import random, randint
+        from mlflow import log_metric, log_param, log_artifacts
+
+        if __name__ == "__main__":
+            # Log a parameter (key-value pair)
+            log_param("config_value", randint(0, 100))
+
+            # Log a metric; metrics can be updated throughout the run
+            log_metric("accuracy", random() / 2.0)
+            log_metric("accuracy", random() + 0.1)
+            log_metric("accuracy", random() + 0.2)
+
+            # Log an artifact (output file)
+            if not os.path.exists("outputs"):
+                os.makedirs("outputs")
+            with open("outputs/test.txt", "w") as f:
+                f.write("hello world!")
+            log_artifacts("outputs")
+
+How do I view my MLFlow runs and experiments?
+---------------------------------------------
+
+Once you've run your code, you may view the results with MLFlow's tracking UI. To start the UI, run:
+
+.. code-section::
+
+    .. code-block:: shell
+
+        mlflow ui
+
+And then navigate to http://localhost:5000 in your browser. You will see a page similar to:
+
+.. image:: _static/images/quickstart-ui-screenshot.png
+    :width: 800px
+    :align: center
+
+You are in the **Default** experiment, which now contains the tracking data for your run. You can view the parameters, metrics, and artifacts of your run by clicking on the run name. 
```

Comment:
Maybe a short sentence following this explaining an experiment is a collection of runs. Can place it between 1 and 2

Reply:
Off-topic technical question: Is it best for an experiment to be a collection of runs with *identical* code (except for configuration values) or is it just "trying to get such-and-such a prediction from the domain, from day 1's terrible architecture to the super-bespoke architecture of day 1000"?

Reply:
Great question! Definitely the latter - `"trying to get such-and-such a prediction from the domain, from day 1's terrible architecture to the super-bespoke architecture of day 1000"?`?